src/main/java/com/efficio/etl/web/controller/angular/AngularMapOntologyController.java

```diff
@@ -318,4 +325,24 @@ public FileUploadForm getForm() {
 		return new FileUploadForm();
 	}
 
+	private void validateTrialInstanceValue(final List<MeasurementVariable> conditions, final List<MeasurementVariable> trialVariables, final Map<String, List<Message>> errorMessages) {
+		for (final MeasurementVariable varCondition : conditions) {
+			if (varCondition.getTermId() == 0) {
+				final List<MeasurementVariable> trialVariable = trialVariables.stream().filter(mVar -> {
+					return mVar.getScale().equals(varCondition.getScale()) && mVar.getMethod().equals(varCondition.getMethod()) && mVar.getProperty().equals(varCondition.getProperty());
+				}).collect(Collectors.toList());
+				if (trialVariable != null && !trialVariable.isEmpty()) {
+					varCondition.setTermId(trialVariable.get(0).getTermId());
```

Comment:
Why do we copy the termId from trialVariables to condition?  I think getTrialVariables is just derived from Workbook.conditions?

Reply:
Hi @abatac , 
Logic added: set measurementid in condition variables